apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -108,36 +129,31 @@ export function InstructorAssessmentSettings({
                   <input
                     type="text"
                     class="form-control"
+                    res.locals.course.path,
                     id="module"
                     name="module"
-                    value="${resLocals.assessment_module
-                      ? resLocals.assessment_module.heading
-                      : ''}"
-                    disabled
+                    list="modulesList"
+                    value="${resLocals.assessment_module.name}"
                   />
+                  ${modulesList(modules)}
                   <small class="form-text text-muted">
                     The <a href="${resLocals.urlPrefix}/course_admin/modules">module</a> this
                     assessment belongs to.
                   </small>
                 </div>
                 <div class="form-group">
                   <label for="aid">AID</label>
-                  ${resLocals.authz_data.has_course_permission_edit &&
-                  !resLocals.course.example_course
-                    ? ChangeIdButton({
-                        label: 'AID',
-                        currentValue: resLocals.assessment.tid,
-                        otherValues: tids,
-                        csrfToken: resLocals.__csrf_token,
-                      })
-                    : ''}
                   <input
                     type="text"
                     class="form-control"
```

Comment:
Consider adding `text-monospace` to this input to help emphasize that it's not just a freeform string.

Reply:
Good idea. Added `text-monospace` in [9173a63](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/9173a63860cec149d3796686882b0ec384a3dc16).